samples/04_gis_analysts_data_scientists/forecasting_monthly_rainfall_in_California_using_deeplearning_timeseries_model_from_arcgis_learn.ipynb

```diff
@@ -0,0 +1,2227 @@
+{
```

Comment:
<p>It is possible that portal might have multiple items named precipitation, so for making this step more robust use url method instead of gis.content.search method.</p><p> </p><pre class="ql-syntax" spellcheck="false">from arcgis.features import FeatureLayer, FeatureLayerCollection
url = 'https://services7.arcgis.com/JEwYeAy2cc8qOe3o/arcgis/rest/services/cali_precipitation/FeatureServer/0'
table = FeatureLayer(url)
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/900/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
<p>this is added as a comment in case of data duplication . The search method is retained since the user is familiar with this method. But the comment ahs been added to show the data access using the url method  </p>